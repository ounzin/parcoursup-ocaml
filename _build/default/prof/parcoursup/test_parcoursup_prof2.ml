(*
fonctionnalité testée:
""""
    Si on appelle deux fois ajoute_voeu pour un même candidat et une même formation,
    c'est le rang utilisé au second appel qui est retenu 
""""
*)

open Test_parcoursup_utils
let session = nouvelle_session ()

let () = 
  ajoute_candidat session ~nom_candidat:"a";
  ajoute_formation session ~nom_formation:"A" ~capacite:1;
  ajoute_formation session ~nom_formation:"B" ~capacite:1;
  ajoute_commission session ~nom_formation:"A" ["a"];
  ajoute_commission session ~nom_formation:"B" ["a"];
  ajoute_voeu session ~rang_repondeur:(Some 1) ~nom_candidat:"a" ~nom_formation:"A";
  ajoute_voeu session ~rang_repondeur:(Some 2) ~nom_candidat:"a" ~nom_formation:"B";
  ajoute_voeu session ~rang_repondeur:(Some 3) ~nom_candidat:"a" ~nom_formation:"A";
  reunit_commissions session;
  nouveau_jour session;
  affiche_propositions_en_attente session "a";
  affiche_voeux_en_attente session "a"