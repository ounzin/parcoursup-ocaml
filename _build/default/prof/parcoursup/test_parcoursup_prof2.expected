nouvelle session
ajoute candidat a
ajoute formation A (1 places)
ajoute formation B (1 places)
commission de classement de A : [a]
commission de classement de B : [a]
ajoute voeu de a pour A (rang=1)
ajoute voeu de a pour B (rang=2)
ajoute voeu de a pour A (rang=3)
r√©unit commissions
nouveau jour
Propositions en attente de a : {B}
Voeux en attente de a : {}
